### DISCUSSION TITLE
problem after upgrade to 1.2.0

### DISCUSSION THREAD

**Discussion by mrbankonly** [qgustavor/mega#185] [General]
It happened once I initialize it (most of the time)
and also happen when using `.download` stream
```
TypeError: fetch failed
node:internal/deps/undici/undici:12345:11
```

But I got this error only in ubuntu server it works fine on a mac both have same Node version: v20.11.1
I was using mega with megajs to stream file

**Comment by qgustavor** [comment]
https://github.com/qgustavor/mega/discussions/138
